cdap-watchdog-api/src/main/java/co/cask/cdap/api/metrics/MetricStore.java

```diff
@@ -43,8 +36,7 @@
    * @return time series that satisfy the query
    * @throws Exception
    */
-  // todo: expose metric-specific APIs instead of Cube ones
```

Comment:
Cube is a generic dataset. MetricStore uses it to store metrics. We want to expose metric-specific apis, as they are slightly different. While keeping Cube apis clean, so that it is highly re-usable